src/include/mpir_contextid.h

```diff
@@ -48,10 +48,8 @@ typedef uint16_t MPIR_Context_id_t;
 #define MPIR_CONTEXT_SUFFIX_WIDTH (1)
 #define MPIR_CONTEXT_SUFFIX_SHIFT (0)
 #define MPIR_CONTEXT_SUFFIX_MASK ((1 << MPIR_CONTEXT_SUFFIX_WIDTH) - 1)
-#define MPIR_CONTEXT_INTRA_PT2PT (0)
-#define MPIR_CONTEXT_INTRA_COLL  (1)
-#define MPIR_CONTEXT_INTER_PT2PT (0)
-#define MPIR_CONTEXT_INTER_COLL  (1)
+#define MPIR_CONTEXT_PT2PT_OFFSET 0
```

Comment:
did you mean to use this in all the places that now use `0` directly?

Reply:
Initially I used this symbol, but then I find it is much easier to read to just use `0`, as it is the default context. What do you think?

Reply:
In most places where we use `MPIR_CONTEXT_INTRA_PT2PT`, it is for `attr` parameter anyway, which the default `0` makes more sense than `MPIR_CONTEXT_PT2PT_OFFSET`.

Reply:
I agree that `MPIR_CONTEXT_INTRA_PT2PT` and its predecessors are awfully verbose for a default parameter. I think I can live with `0`.

Reply:
@raffenet Is this PR good?

Reply:
Oops, yes. Will approve.